### ISSUE TITLE
Stop Gradient in model

### ISSUE THREAD

**Issue by SeaOfOcean** [intel-analytics/BigDL#1033]
There is an example in keras

[stop_gradient](https://keras.io/backend/)

```
stop_gradient(variables)
```
Returns variables but with zero gradient w.r.t. every other variable
